_data/marynaggita/README.md

```diff
@@ -0,0 +1,220 @@
+# [marynaggita](https://github.com/marynaggita/)'s Zarr Collection
+
+## Tweet Mentions
+
+- https://twitter.com/latlong_blog/status/1543726888908009472
+- https://twitter.com/_jhamman/status/1579882167303168004
+- https://twitter.com/ErikvanSebille/status/1578030942144188417
+
+
+
+## Zarr Data Visualisation
+
+![image1](https://github.com/marynaggita/beautiful-zarr/blob/marynaggita/_data/marynaggita/Screenshots/data3.png)
+
+Use Case: Study Amazon Estuaries with Data from the EOSDIS Cloud(https://github.com/podaac/tutorials/blob/master/notebooks/SWOT-EA-2021/Estuary_explore_inCloud_zarr.ipynb)
```

Comment:
Is this the description of first image or the second one?

Reply:
> Is this the description of the first image or the second one?

the description is for the second one.